src/components/List/List.jsx

```diff
@@ -64,6 +74,27 @@ const List = () => {
 
   const clearFilter = () => setInputValue('');
 
+  const handleDelete = (id) => {
+    const clear = () => {
+      clearTimeout(timeSpecial);
+    };
+
+    if (window.confirm('are you sure to delete the item?')) {
+      setLoad('Removing element...');
+      remove(token, id)
+        .then(() => {
+          setLoad('');
+          setSuccess('Document successfully deleted!');
```

Comment:
Maybe using a unique name (item, element or document) in the different messages can give more clarity to the user

Reply:
it has been solved!!